examples/fluids/index.md

```diff
@@ -13,22 +13,22 @@ Moreover, the Navier-Stokes example has been developed using PETSc, so that the
 ```
 ## The Navier-Stokes equations
 
-The mathematical formulation (from {cite}`giraldoetal2010`, cf. SE3) is given in what follows.
+The mathematical formulation (from {cite}`shakib1991femcfd`) is given in what follows.
 The compressible Navier-Stokes equations in conservative form are
 
 $$
 \begin{aligned}
 \frac{\partial \rho}{\partial t} + \nabla \cdot \bm{U} &= 0 \\
-\frac{\partial \bm{U}}{\partial t} + \nabla \cdot \left( \frac{\bm{U} \otimes \bm{U}}{\rho} + P \bm{I}_3 -\bm\sigma \right) + \rho g \bm{\hat k} &= 0 \\
-\frac{\partial E}{\partial t} + \nabla \cdot \left( \frac{(E + P)\bm{U}}{\rho} -\bm{u} \cdot \bm{\sigma} - k \nabla T \right) &= 0 \, , \\
+\frac{\partial \bm{U}}{\partial t} + \nabla \cdot \left( \frac{\bm{U} \otimes \bm{U}}{\rho} + P \bm{I}_3 -\bm\sigma \right) - \rho \bm{b}  &= 0 \\
+\frac{\partial E}{\partial t} + \nabla \cdot \left( \frac{(E + P)\bm{U}}{\rho} -\bm{u} \cdot \bm{\sigma} - k \nabla T \right) - \rho \bm{b} \cdot \bm{u} &= 0 \, , \\
 \end{aligned}
 $$ (eq-ns)
 
 where $\bm{\sigma} = \mu(\nabla \bm{u} + (\nabla \bm{u})^T + \lambda (\nabla \cdot \bm{u})\bm{I}_3)$ is the Cauchy (symmetric) stress tensor, with $\mu$ the dynamic viscosity coefficient, and $\lambda = - 2/3$ the Stokes hypothesis constant.
-In equations {eq}`eq-ns`, $\rho$ represents the volume mass density, $U$ the momentum density (defined as $\bm{U}=\rho \bm{u}$, where $\bm{u}$ is the vector velocity field), $E$ the total energy density (defined as $E = \rho e$, where $e$ is the total energy), $\bm{I}_3$ represents the $3 \times 3$ identity matrix, $g$ the gravitational acceleration constant, $\bm{\hat{k}}$ the unit vector in the $z$ direction, $k$ the thermal conductivity constant, $T$ represents the temperature, and $P$ the pressure, given by the following equation of state
+In equations {eq}`eq-ns`, $\rho$ represents the volume mass density, $U$ the momentum density (defined as $\bm{U}=\rho \bm{u}$, where $\bm{u}$ is the vector velocity field), $E$ the total energy density (defined as $E = \rho e$, where $e$ is the total energy including thermial and kinetic but not potential energy), $\bm{I}_3$ represents the $3 \times 3$ identity matrix, $\bm{b}$ is a body force vector (e.g., gravity vector),  $k$ the thermal conductivity constant, $T$ represents the temperature, and $P$ the pressure, given by the following equation of state
```

Comment:
```suggestion
In equations {eq}`eq-ns`, $\rho$ represents the volume mass density, $U$ the momentum density (defined as $\bm{U}=\rho \bm{u}$, where $\bm{u}$ is the vector velocity field), $E$ the total energy density (defined as $E = \rho e$, where $e$ is the total energy including thermial and kinetic but not potential energy), $\bm{I}_3$ represents the $3 \times 3$ identity matrix, $\bm{b}$ is a body force vector (e.g., gravity vector, $\rho \bm{g} z$),  $k$ the thermal conductivity constant, $T$ represents the temperature, and $P$ the pressure, given by the following equation of state
```

Reply:
First, I see I miss spelled thermal (no i).  As for the suggested change,   there is no z there.
\bm{b} = \bm{g}.   It is fairly traditional (in aero and atmospheric flows)  to choose a coordinate system where the earths surface is x, y and thus g=0 0 -9.81 (as we apply in the DC inputs)  and then the ma=F of NS has a  -9.81*rho "forcing", F on m a_3. Shakib puts script F on the RHS of the equation and has it = rho{0, b_i, b_i u_i} and here b_3=-9.81.

Where this gets tricky is that in a residual, we pull it to the same side of equation and call it - script F which means momentum terms get -b_i, energy -b_i u_i.   As I was reviewing code I was seeing lots of negations of g.   I think the existing docs has -S instead of -F but I will take a look at that after I send this comment (learned the hard way a few times over the last couple of days that when I leave this page my comment disappears)

Reply:
After reviewing the docs, I am fine with dropping the i from thermial to make thermal and just dropping " z" from your suggestion.  Care with the sign of components of g_i is always part or setting up problems properly and, as Evans accurately points out, verbosity in the attempt to be "utterly clear" is sometimes working against its objective and simple is often better.